docs/source/support/compatibility.rst

```diff
@@ -72,6 +72,7 @@ Summary of Ledger API Changes
        | Remove Reset Service
        | Deprecate Ledger Identity Service
        | Make ledger_id and application_id fields optional
+       | Error codes returned by the gRPC services have changed (originally a run-time option since the Daml Connect SDK 1.18.0)
```

Comment:
Did the error codes change for specific requests or did we drop the feature flag from the CLI? I am not sure what's supposed to be outlined here

Reply:
Less is more, let me drop the text in parentheses.